app/src/app/home/pages/batch/batch.service.ts

```diff
@@ -320,6 +320,8 @@ export class BatchPageService extends IPageService<BatchPageItem> {
         if (!params.dstBatchSlug) {
           params.dstBatchSlug = this.cache.batch.comparedAgainst;
         }
+        params.dstBatchName = params.dstBatchSlug.split('@')[0];
+        params.srcBatchName = params.srcBatchSlug.split('@')[0];
```

Comment:
What do these lines do?

Reply:
Oh. That's because the existing implementation includes a special treatment of versions that are of the form "v1.2.3@platform-arch-compiler-version-etc" that are used by one of our customers. Let's not remove these two lines please. I'm worried that we would unintentionally break their use-case somehow.

Reply:
If it is not absolutely necessary, let's take the side of caution and not change the existing behavior for the upcoming release.